### ISSUE TITLE
Mutliple variants causing problems

### ISSUE THREAD

**Issue by LarBearrr** [SaifurRahmanMohsin/MobileUser-Plugin#3]
I went to create a new variant and received duplicate index error regarding my package name, but only one variant was listed in the app.

I went into phpmyadmin and cleared out the tables and it allowed me to create the variant. However, when I went to the Mobile controller to look at my installed instances, I received an out of bounds index error. 

I looked into the code and found the problem and had to clear out the sessions table from the database.

I neglected to take some screenshots so my explanation may not be very helpful.

Update: Here's a screen shot
![image](https://user-images.githubusercontent.com/28734844/39103212-089073b8-466f-11e8-83f2-5fdd58056d51.png)

**Comment by SaifurRahmanMohsin** [comment]
Hi. Thank you for reporting this. I’ll be verifying this and pushing an update to having this fixed. Will close the issue once patched and an update is pushed.

**Comment by WWRFresource** [comment]
Sorry to keep bothering you. I am trying to get your plugins to work with a React Native project.. so far I am able to get my login working but I am having trouble with the Install API.

 

Using Postman, I am able to post to the end point and I think I have narrowed down my problem to around Line 86 where the method tries to save a new Install:

 

if($install->save()) {

 

I did some debugging by placing a return statement throughout the method (return $instance_id) and was able to get it to return properly all the way up to just before this line.

 

Now, I changed the save() method to create() and it will return $instance_id. If I removed the debug statement, show the install message in Postman.

 

However, going back to my app, if try to run call the Install endpoint there, I receive an Unrecognized token ‘<’ which tells me the JSON response is not correct.

 

What I’m trying to figure out is why Postman would be returning a JSON object but not my app.

 

Thanks again, I hope to hear back from you.

 

Larry
